content/docs/releasing.md

```diff
@@ -62,23 +60,22 @@ When the project has been released as a public project on GitHub the following w
 
 - Semantically version project artifacts. You MUST tag all versions in GitHub with the exact version name: e.g., 0.1.0
 - Make sure that no credentials, private identifiers or personal data is at any time present in your repository
-- Make sure code-reviews with by least 2 Saucers on all code to minimize the risk of implanted security backdoors and vulnerable code.
+- Make sure code-reviews with by least 1 Sauce employee on all code to minimize the risk of implanted security backdoors and vulnerable code.
 - Make sure there is an active team of maintainers of at least 2 Saucers taking ownership of the project
-- Make sure there is a continuous integration setup in place that runs all tests and measures coverage for every commit in every branch and in every pull request
-- Make sure there is a code analysis tool (like Codacy or SonarQube) checking the code health for every commit in every branch and also for all pull requests
+- Make sure there is a continuous integration setup in place that runs all linting, tests and measures coverage for every commit in every branch and in every pull request
```

Comment:
I would prefer to leave it as it was, mentioning the code analysis should be explicit and we should not assume people will include as part of their pipeline.